distributions/LibreELEC/kernel_options

```diff
@@ -25,3 +25,33 @@ CONFIG_ASYMMETRIC_PUBLIC_KEY_SUBTYPE=y
 CONFIG_X509_CERTIFICATE_PARSER=y
 CONFIG_PKCS7_MESSAGE_PARSER=y
 CONFIG_PKCS8_PRIVATE_KEY_PARSER=m
+
+# packages/sysutils/systemd
+CONFIG_DEVTMPFS=y
+CONFIG_CGROUPS=y
+CONFIG_INOTIFY_USER=y
+CONFIG_SIGNALFD=y
+CONFIG_TIMERFD=y
+CONFIG_EPOLL=y
+CONFIG_NET=y
+CONFIG_SYSFS=y
+CONFIG_PROC_FS=y
+CONFIG_FHANDLE=y
+CONFIG_CRYPTO_USER_API_HASH=y
+CONFIG_CRYPTO_HMAC=y
+CONFIG_CRYPTO_SHA256=y
+CONFIG_SYSFS_DEPRECATED=n
+CONFIG_UEVENT_HELPER=n
+CONFIG_UEVENT_HELPER_PATH=n
+CONFIG_FW_LOADER_USER_HELPER=n
+CONFIG_IPV6=y
+CONFIG_AUTOFS4_FS=y
+CONFIG_TMPFS_XATTR=y
+CONFIG_TMPFS_POSIX_ACL=y
+CONFIG_EXT4_FS_POSIX_ACL=y
+CONFIG_BTRFS_FS_POSIX_ACL=y
+CONFIG_SECCOMP=y
+CONFIG_SECCOMP_FILTER=y
+CONFIG_CHECKPOINT_RESTORE=y
```

Comment:
The systemd README mentions this is only needed to get kcmp, that line was added 6 years ago maybe KCMP option wasn't individually selectable by then.

Since 5.10.20 DRM selects KCMP so this can probably be just dropped

Reply:
the extra check doesn't hurt

Reply:
then I'd suggest switch that to `CONFIG_KCMP` as this is what systemd really seems to want

Reply:
`CONFIG_KCMP` was added quite recently and made it into stable 5.10.20 https://github.com/gregkh/linux/commit/1ea36020950d44

As we'll be using 5.10.20 or newer on all devices soon I'd go for that instead of `CONFIG_CHECKPOINT_RESTORE`

Reply:
heh, sure